docs/changehistory/NextVersion.md

```diff
@@ -81,6 +81,25 @@ It is frequently necessary to detect the type of JavaScript process currently ex
 
 ## Common table expression support in ECSQL
 
+Planar clip masks provide a two and a half dimensional method for masking the regions where the background map, reality models and BIM geometry overlap. A planar clip mask is described by [PlanarClipMaskProps]($common).A planar clip mask may be applied to a contexual reality model as a [ContextRealityModelProps.planarClipMask]($common) to the background map as [BackgroundMapProps.planarClipMask]($common) or as an override to attached reality models with the [DisplayStyleSettingsProps.planarClipOvr]($common) array of [DisplayStyleRealityModelPlanarClipMaskProps]($common).   The planar clip mask geometry is not required to be planar as the masks will be generated from their projection to the X-Y plane, therefore any 3D model or reality model can be used to generate a planar clip mask.
```

Comment:
Move your additions to their own section - they are in the middle of the Common Table Expressions notes.

Reply:
Fixed